python/GafferSceneTest/GroupTest.py

```diff
@@ -777,6 +777,22 @@ def testProcessInvalidSet( self ) :
 
 		self.assertSceneValid( group["out"] )
 
+	def testExistsAndSortedChildNames( self ) :
```

Comment:
Name feels a bit weird.  If sortedChildNames is a private plug which doesn't need to be explicitly tested because we only interact with it through exists, it probably shouldn't be mentioned in the name of the test?

Reply:
Thanks for spotting that - fixed in 7799022. I'd written that test at a time when `sortedChildNamesPlug()` was public, and when I removed the checks for it I failed to update the function name at the same time.